scripts/remove_after_use/node_preprint_es.py

```diff
@@ -0,0 +1,47 @@
+import django
+django.setup()
+import sys
+import logging
+from django.db import transaction
+
+from scripts import utils as script_utils
+from website import search
+from website.search.elastic_search import delete_doc
+from osf.models import Preprint, AbstractNode
+import progressbar
+logger = logging.getLogger(__name__)
+
+# To run: docker-compose run --rm web python -m scripts.remove_after_use.node_preprint_es
+def main(dry=True):
+    """
+    Temporary script for updating elastic search after the node-preprint divorce
+    - Removes nodes from the index that are categorized as preprints
+    - Adds these nodes to the index, this time categorized as nodes
+    - Adds preprints to the index, categorized as preprints
+    """
+    preprints = Preprint.objects
+    progress_bar = progressbar.ProgressBar(maxval=preprints.count()).start()
+
+    for i, p in enumerate(preprints.all(), 1):
+        progress_bar.update(i)
+        logger.info('Adding preprint {} to index.'.format(p._id))
+        if not dry:
+            search.search.update_preprint(p, bulk=False, async=False) # create new index for preprint
+        if p.node:
+            logger.info('Deleting node {} from index, with category preprint'.format(p.node._id))
```

Comment:
Nit: you can probably get rid of all the logging. Otherwise, the progress bar won't be visible. The node _id's aren't really necessary to log anyway, since we can easily re-construct the the set of affected nodes (all nodes attached to a preprint).